src/main/java/com/sandvoxel/generitech/common/tileentities/power/TestPower.java

```diff
@@ -26,195 +27,44 @@
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
 import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.energy.IEnergyStorage;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraftforge.energy.IEnergyStorage'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)